app-editors/featherpad/featherpad-1.0.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/FeatherPad-${PV}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+X"
```

Comment:
Just a quick note, we try to use `gui` USE flag for this going to the future.

Reply:
Thx, but this flag isn't about GUI or not. From the INSTALL file (and from my own testing):

> If, for whatever reason, you do not want the X11 support, you could first turn on the option "WITHOUT_X11" with cmake:
> [...]
> The result will have all features except for virtual desktop awareness and tab DND.

I think using "gui" as use flag will be confusing.

Reply:
Lets remain with `X` USE flag, but add an explanation what it does.
Please add to the `metadata.xml` an explanation on the USE flag, so it won't confuse.
You can look at the documentation [here](https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html), and an example [here](https://github.com/gentoo/gentoo/blob/master/app-text/sigil/metadata.xml#L8-L11).

Reply:
Yeah, sorry, mistook it toggled GUI with a configure option `--with-X` like many other programs do. Glad you checked upstream!

Reply:
Have added use flag description.